[Bug] npx plotlink-ows crashes: Prisma client not generated on install

## Problem

Fresh `npx plotlink-ows@latest` crashes immediately with:

```
Error: Cannot find module '../node_modules/.prisma/local-client/index.js'
```

The Prisma client (`@prisma/client`) needs `prisma generate` to create the client files, but this never runs during `npm install` / `npx` install.

## Root Cause

No `postinstall` script in `package.json` to run `prisma generate --schema app/prisma/schema.prisma`.

## Fix

Add a `postinstall` script to `package.json`:

```json
"postinstall": "prisma generate --schema app/prisma/schema.prisma"
```

This runs automatically after `npm install` and when npx installs the package.

**Alternative:** If Prisma generate is too slow for postinstall, generate the client at startup in `bin/plotlink-ows.js` before spawning the server (check if `.prisma/local-client` exists, if not, run generate).

## Acceptance Criteria

- [ ] `npx plotlink-ows@latest` works on a fresh install without manual steps
- [ ] Prisma client is generated automatically
- [ ] No `MODULE_NOT_FOUND` crash on first launch

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "plotlink-ows",
-  "version": "1.0.1",
+  "version": "1.0.2",
   "bin": {
     "plotlink-ows": "./bin/plotlink-ows.js"
   },
@@ -26,6 +26,7 @@
     "app:dev": "concurrently \"tsx watch app/server.ts\" \"vite --config app/vite.config.ts\"",
     "app:build": "vite build --config app/vite.config.ts",
     "app:start": "tsx app/server.ts",
+    "postinstall": "prisma generate --schema app/prisma/schema.prisma",
     "prisma:local": "prisma generate --schema app/prisma/schema.prisma && prisma db push --schema app/prisma/schema.prisma",
     "release:patch": "npm version patch && git push origin main --follow-tags && VERSION=$(node -p 'require(\"./package.json\").version') && gh release create \"v$VERSION\" --generate-notes --latest && npm publish",
     "release:minor": "npm version minor && git push origin main --follow-tags && VERSION=$(node -p 'require(\"./package.json\").version') && gh release create \"v$VERSION\" --generate-notes --latest && npm publish",

